grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -262,6 +262,7 @@ class CycleCountService {
         cycleCounts.each { CycleCountDto cycleCount ->
             cycleCount.cycleCountItems.each { CycleCountItemDto item ->
                 data << [
+                        "Cycle count id": cycleCount.id,
```

Comment:
I'm not a huge fan of needing to put cycle count id in the xls file since I'd rather not expose our internal ids to users. Can the frontend not look up the count by product? If this is the only/best solution, we should discuss with Manon to confirm that it's okay to include it.